spyder/config/user.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright © Spyder Project Contributors
+# Copyright © 2009-2013 Pierre Raybaut
+# Copyright © 2014-2015 The Spyder Development Team

```

Comment:
Please leave

    Spyder Project Contributors

and remove the next two lines instead.